src/authentication/Authentication.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
```

Comment:
This component is displayed when the user needs to log in. Right now it switches between the "Welcome" screen and the "Log in" screen, but soon it will host all the other steps for registering a new cozy account.